scripts/al2022/install-workarounds.sh

```diff
@@ -1,17 +1,6 @@
 #!/usr/bin/env bash
 set -ex
 
-# update to AL2022 release that has latest repos
-# (get the latest version from dnf check-release-update)
-INFOFILE=$(mktemp)
-trap "rm -f $INFOFILE" EXIT
-
-sudo dnf check-release-update >$INFOFILE 2>&1
-RELEASESERVER=$(cat $INFOFILE | grep "releasever*" | tail -1 | awk '{print $3}')
-if [ -n "$RELEASESERVER" ]; then
-    sudo dnf update -y $RELEASESERVER
-fi
-
 # install ssm (required until amazon-ssm-agent package is available in al2022 repos)
```

Comment:
is amazon-ssm-agent still not available in al2022 repos?

Reply:
yeah, it is still not ready for AL2022 yet.